sdk/core/openrewrite/README.md

```diff
@@ -0,0 +1,102 @@
+# Azure Code Migration with OpenRewrite
```

Comment:
We should not have openrewrite package in `sdk/core` directory. It should be in `sdk/tools` directory.